Explicitly state that we're not creating config file (#3310)
alacritty/alacritty: README.md

You can find the default configuration file with documentation for all available
fields on the [GitHub releases page](https://github.com/alacritty/alacritty/releases) for each release.

Alacritty doesn't create config file for you, therefore you should put it in one

> of the following locations yourself:

Reviewer: I don't think the old meaning implicated that the configuration file is created by Alacritty, but it did make it more clear that there's a fallback priority here.

Our manpage also still states this exact thing.

It might be easier to just change the previous paragraph from `You can find the default configuration file...` to `You can copy the default configuration file...`.

Author: I feel like this part `Alacritty doesn't create config file for you` is important tbh. Like you can't just rely on `copy` tbh. Like I could understand the idea behind copy, but you shouldn't judge people by yourself here.

`Alacritty doesn't create config file for you, if you want to tune it for your needs create one in one of the following locations:`

?

Reviewer: It should be "doesn't create **a** config file for you" or *the* works too I suppose.

 The `one in one` part also doesn't really flow very well and I'm not sure if `tune` is the right word, make `tweak` is a bit better? And I think it should be `to your needs`, rather than `for your needs`.

Maybe we should just keep it a bit simpler like this:

```
Alacritty doesn't create the config file for you, but it looks for it in the following locations:
```

Should still be very clear, makes it obvious that there is a fallback behavior going on and it doesn't include any text besides what is necessary. The biggest problem is that the `but` might not be optimal, but it should be fine?

Reviewer 2: I'm a fan of @chrisduerr's wording above. Maybe change `it looks for it in` -> `it looks for one in`, though.

The `but` seems fine to me. Could also go for the super intellectual `...for you; however, it looks for one...`. Instant bonus points for using a semicolon.

### Windows

> On Windows the config file should be at:

Reviewer 2: ```suggestion
On Windows, the config file should be located at:
```

> On Windows the config file should be put in:

Reviewer: ```suggestion
On Windows the config file should be at:
```

We're directly linking a file location, not a directory. So we shouldn't use 'in'.
